test/e2e/kubernetes/kubernetes_test.go

```diff
@@ -798,16 +809,6 @@ var _ = Describe("Azure Container Cluster using the Kubernetes Orchestrator", fu
 				valid := s.Validate("(Welcome to nginx)", 5, 30*time.Second, cfg.Timeout)
 				Expect(valid).To(BeTrue())
 
-				By("Ensuring we have outbound internet access from the nginx pods")
```

Comment:
We're already doing this test in the long-running php-apache deployment (used for HPA tests and pod-to-pod connectivity tests)